packages/node/test/integration/secure-withdraw.spec.ts

```diff
@@ -17,6 +18,8 @@ import {
   transferERC20Tokens
 } from "./utils";
 
+expect.extend({ toBeEq });
```

Comment:
Why not `extend` in the global setup and replace all `BigNumber` comparison instances as done below?